eng/targets/DiaSymReaderNative.targets

```diff
@@ -0,0 +1,21 @@
+<Project>
```

Comment:
Nit: Missing copyright header.

I'd add a comment that describes what the file is for.

Reply:
Added both. Looks like we missed copyrights in some other files so added those as well.